language/move-lang/src/parser/syntax.rs

```diff
@@ -863,11 +828,12 @@ fn parse_dot_chain<'input>(tokens: &mut Lexer<'input>) -> Result<Exp, Error> {
 // Parse a base type:
 //      BaseType =
 //          <ModuleAccess>
-//          | <ModuleAccessBeginArgs> <TypeArgs>
+//          | <ModuleAccess> "<" <TypeArgs>
 fn parse_base_type<'input>(tokens: &mut Lexer<'input>) -> Result<SingleType, Error> {
     let start_loc = tokens.start_loc();
-    let (tn, has_args) = parse_module_access(tokens)?;
-    let tys = if has_args {
+    let tn = parse_module_access(tokens)?;
+    let tys = if tokens.peek() == Tok::Less {
```

Comment:
Does this allow spaces? We could allow it in types now I think

Reply:
Yes, spaces no longer matter, except in the one case of an identifier followed by "<" in the context of an expression.